src/layouts/EditNavBar.jsx

```diff
@@ -724,10 +724,7 @@ const EditNavBar = ({ match }) => {
               )}
               <LoadingButton
                 isDisabled={hasErrors()}
-                onClick={async () => {
-                  await saveNavData({ originalNav, links, sha })
-                  window.location.reload()
-                }}
+                onClick={async () => saveNavData({ originalNav, links, sha })}
```

Comment:
should we still `await` here?

Reply:
There's no need for this because it's effectively 
```
async () => {
  return saveNavData(...)
}
```